doc: Add Dragino LHT52 (#1240)
TheThingsIndustries/lorawan-stack-docs: doc/content/devices/models/dragino-lht52/_index.md

---
title: "Dragino LHT52"
description: ""
weight: 
---

{{< figure src="lht52.png" alt="Generic Node Sensor Edition" class="float plain">}}

> The Dragino [LHT52](https://www.dragino.com/products/temperature-humidity-sensor/item/199-lht52.html) is a temperature & humidity sensor. It features a Type-C port for connecting to accessories, such as an external temperature sensor for example.

Reviewer: @LDannijs It should be an external temperature probe. 
It supports AS-01 temperature probe (https://www.aliexpress.com/item/1005004789167071.html)

> The Dragino [LHT52](https://www.dragino.com/products/temperature-humidity-sensor/item/199-lht52.html) is a temperature & humidity sensor. It features a Type-C port for connecting to accessories, such as an external temperature sensor for example.

Author: Correct. It's not a traditional USB port where its used for charging, but instead as a means to connect to, for example, external probes. For more info [see here too](http://wiki.dragino.com/xwiki/bin/view/Main/User%20Manual%20for%20LoRaWAN%20End%20Nodes/LHT52%20-%20LoRaWAN%20Temperature%20%26%20Humidity%20Sensor%20User%20Manual/#H5.A0SensorsA0andAccessories)

<!--more-->

## Provisioning Information

Provisioning information (The **JoinEUI/AppEUI**, **DevEUI** and **AppKey**) can be found on a sticker inside the box. 

{{< figure src="lht52-prov-info.png" alt="LHT52 provisioning information" width="50%">}}

> {{< note "The LHT52 does not support onboarding with a QR code. You can onboard it in the section below." />}}

Author: It does have a QR code, but it is not used for onboarding.

## Onboarding to {{% tts %}}

The device can be onboarded using the Device Repository or by manually entering the specific information.

> The [LoRaWAN® Device repository](https://github.com/TheThingsNetwork/lorawan-devices) comprises over 600 end-device profiles, including the Dragino LHT52, which enables quick and easy registration with {{% tts %}}.

Reviewer 2: ```suggestion
The [LoRaWAN® Device repository](https://github.com/TheThingsNetwork/lorawan-devices) comprises over 600 end-device profiles, including the Dragino LHT52, which enables quick and easy onboarding to {{% tts %}}.
```

In the **End device type** section, under **Input method**, select either the **Select the end device in the LoRaWAN Device Repository** or **Enter end device specifics manually** option. The [Onboarding devices without QR codes]({{< ref "/devices/adding-devices/#onboarding-devices-without-qr-codes" >}}) section of the [Adding Devices]({{< ref "/devices/adding-devices/" >}}) guide explains this procedure in detail. Following details are LHT52-specific:

- End device brand: **Dragino Technology Co., Limited**
- Model: **LHT52**
- Hardware Ver.: **Unknown ver.**
- Firmware Ver.: **1.0**

{{< note >}} If you are manually adding the device, enter these details specifically for the LHT52:
- LoRaWAN version: **LoRaWAN Specification 1.0.3**
- Regional parameters version: **RP001 Regional Parameters 1.0.3 revision A** {{</ note >}}

> After configuring your device, select the **Register end device** button.

Reviewer 2: Pls take a look at @pradeeka7's latest PRs for how to edit this section. Instructions for "manually" shouldn't be under {{< note >}} tags

---
title: "Dragino LHT52"
description: ""
weight: 
---

{{< figure src="lht52.png" alt="Generic Node Sensor Edition" class="float plain">}}

> The Dragino [LHT52](https://www.dragino.com/products/temperature-humidity-sensor/item/199-lht52.html) is a temperature & humidity sensor and has a USB Type-C port to connect to external sensors such as an external temperature sensor.

Reviewer 2: ```suggestion
The Dragino [LHT52](https://www.dragino.com/products/temperature-humidity-sensor/item/199-lht52.html) is a temperature & humidity sensor. It features a Type-C port so it can be connected as an external temperature sensor.
```

<!--more-->

> # Onboarding to {{% tts %}}

Reviewer 2: If you deploy this doc locally, you can see that if you use #, this title won't be visible on the right hand menu. We want every title in the doc to be shown so please replace # here with ## and adjust all other titles accordingly (replace ## with ###, etc.)

> The provisioning information, which are the **JoinEUI**, **DevEUI** and **AppKey** can be found on a sticker inside the box. 

Reviewer 2: ```suggestion
Provisioning information (JoinEUI/AppEUI, DevEUI and AppKey) can be found on a sticker inside the box. 
```

{{< figure src="lht52-prov-info.png" alt="LHT52 provisioning information" width="50%">}}

> The LHT52 does not support onboarding with a QR code. You must proceed directly to the options [**Select the end device in the LoRaWAN Device Repository**](#using-the-lorawan-device-repository) or [**Enter end device specifics manually**](#manually).

Reviewer 2: This is worth being put inside a {{< note >}} tag

> The LHT52 does not support onboarding with a QR code. You must proceed directly to the options [**Select the end device in the LoRaWAN Device Repository**](#using-the-lorawan-device-repository) or [**Enter end device specifics manually**](#manually).

Reviewer 2: ```suggestion
The LHT52 does not support onboarding with a QR code. You can onboard it [**Using the LoRaWAN Device Repository**](#using-the-lorawan-device-repository) or [**Manually**](#manually).
```

> ## Using the LoRaWAN Device Repository

Reviewer 2: I think this and content under Manually can be put under the same title, you can simplify it by just highlighting differences. It's not that this doesn't look good because it does, it just that we already have these titles [here](https://www.thethingsindustries.com/docs/devices/adding-devices/), so if we use the same ones here as well, we will have different content under same titles and by doing that, I think our search engine will eventually start throwing garbage.

The [LoRaWAN® Device repository](https://github.com/TheThingsNetwork/lorawan-devices) comprises over 600 end-device profiles, including the Dragino LHT52, which enables quick and easy registration with {{% tts %}}.

> In the **End device type** section, under **Input method**, select **Select the end device in the LoRaWAN Device Repository** option. The [Onboarding devices without QR codes]({{< ref "/devices/adding-devices/#onboarding-devices-without-qr-codes" >}}) section of the [Adding Devices]({{< ref "/devices/adding-devices/" >}}) page explains all the configurations you need to provide. However, the following are the LHT52 specific configurations you want to include:

Reviewer 2: ```suggestion
In the **End device type** section, under **Input method**, select **Select the end device in the LoRaWAN Device Repository** option. The [Onboarding devices without QR codes]({{< ref "/devices/adding-devices/#onboarding-devices-without-qr-codes" >}}) section of the [Adding Devices]({{< ref "/devices/adding-devices/" >}}) guide explains this procedure in detail. Following details are LHT52-specific:
```

- End device brand: **Dragino Technology Co., Limited**
- Model: **LHT52**
- Hardware Ver.: **Unknown ver.**
- Firmware Ver.: **1.0**

> After providing all the configurations, select the **Register end device** button.

Reviewer 2: ```suggestion
After configuring your device, select the **Register end device** button.
```

{{< figure src="lht52-prov-info-stack.png" alt="Settings for registration through device repository" >}}

## Manually

> The LHT52 also supports manual registration, but make sure all configurations are provided correctly.

Reviewer 2: remove

> In the **End device type** section, under **Input Method**, select **Enter end device specifics manually** option. The [Manually adding devices]({{< ref "/devices/adding-devices/manual/" >}}) and [Over the Air Activation (OTAA)]({{< ref "/devices/adding-devices/manual/otaa/" >}}) pages explain all the configurations you need to provide. However, the following are the LHT52 specific configurations you want to include:

Reviewer 2: ```suggestion
In the **End device type** section, under **Input Method**, select **Enter end device specifics manually** option. The [Manually adding devices]({{< ref "/devices/adding-devices/manual/" >}}) and [Over the Air Activation (OTAA)]({{< ref "/devices/adding-devices/manual/otaa/" >}}) guides explain this procedure in detail. Following details are LHT52-specific:
```

- LoRaWAN version: **LoRaWAN Specification 1.0.3**
- Regional parameters version: **RP001 Regional Parameters 1.0.3 revision A**

> After providing all the configurations, select the **Register end device** button.

Reviewer 2: ```suggestion
After configuring your device, select the **Register end device** button.
```

{{< figure src="manual-register.png" alt="Settings for manual registration" >}}

> ## Monitoring Live Data

Reviewer 2: This title should be 1 size larger than the previous two, i.e. the same size as Onboarding to TTS

In both registration options, after registering the device, you will be redirected to the **Overview** tab of the **End devices** page.

{{< figure src="lht52-overview.png" alt="Overview tab" >}}

> 

Reviewer 2: Remove this, this is pretty straightforward and already mentioned in adding devices doc, so here it's just creating redundancy

> Then, select the **Live Data** tab to view all messages exchanged between the end device and the Network Server.

Reviewer 2: ```suggestion
When your device is registered, select the **Live Data** tab to view all messages exchanged between your end device and {{% tts %}}.
```
